core/src/nodes/handled_node_tasks.rs

```diff
@@ -202,7 +202,7 @@ impl<TInEvent, TOutEvent, THandler> HandledNodesTasks<TInEvent, TOutEvent, THand
         self.tasks.keys().cloned()
     }
 
-    /// Provides an API similar to `Stream`, except that it cannot error.
+    /// Provides an API similar to `Stream`, except that it cannot produce an error.
```

Comment:
I think this was fine as it was?

Reply:
Error isn't a verb. You could say cannot fail, cannot err, cannot produce an error.

Reply:
You are correct, my bad.